Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM swift:3.1
```

Comment:
We can changed to any Linux destrobute that support by swift. Now dockerfile use ubuntu